packages/travesty/travesty.0.7.0/opam

```diff
@@ -0,0 +1,48 @@
+opam-version: "2.0"
+synopsis: "Traversable containers, monad extensions, and more"
+description: """
+'Travesty' is a library for defining containers with applicative
+or monadic traversals, inspired by Haskell's Traversable typeclass; it also
+contains various helpers for monadic code, such as state transformers and
+extension functions for common monads and containers.  It sits on top of Jane
+Street's Base library and ecosystem."""
+maintainer: ["Matt Windsor <mattwindsor91@gmail.com>"]
+authors: ["Matt Windsor <mattwindsor91@gmail.com>"]
+license: "MIT"
+homepage: "https://MattWindsor91.github.io/travesty/"
+doc: "https://MattWindsor91.github.io/travesty/"
+bug-reports: "https://github.com/MattWindsor91/travesty/issues"
+depends: [
+  "dune" {>= "2.0"}
+  "ocaml" {>= "4.08"}
+  "ppx_jane" {>= "v0.12.0" & < "v0.15.0"}
+  "ppx_expect" {with-test & >= "v0.12.0" & < "v0.15.0"}
+  "base" {>= "v0.12.0" & < "v0.15.0"}
+]
+conflicts: [
+  "bisect_ppx" {>= "2.6.0"}
```

Comment:
why is there a conflict with bisect_ppx 2.6.0?

Reply:
brute-force quickfix to stop dependency cycles involving `odoc` and `bisect_ppx` (which were causing my CI to fail and possibly would've caused opam CI to fail too?), but https://github.com/ocaml/opam-repository/pull/18141 _should_ fix them and make the conflict unnecessary.

Besides, now I come to think of it, there's probably a way of specifying a build-time-only conflict, in which case this should've been one of those.